android/src/main/java/com/reactnativehmssdk/HMSManager.kt

```diff
@@ -459,10 +594,74 @@ class HMSManager(reactContext: ReactApplicationContext) :
     }
   }
 
-  /**
-   * Builds and returns PictureInPictureParams as per given config
-   * Currently we are supporting only "aspectRatio" in config
-   */
+  @RequiresApi(Build.VERSION_CODES.O)
+  private fun getPIPRemoteActions(endAction: Boolean = false, audioAction: Boolean = false, videoAction: Boolean = false): MutableList<RemoteAction> {
+    val hmssdk = getHmsInstance()[PipActionReceiver.sdkIdForPIP!!]?.hmsSDK
+
+    pipRemoteActionsList.clear()
+
+    if (audioAction && hmssdk?.getLocalPeer()?.hmsRole?.publishParams?.allowed?.contains("audio") === true) {
```

Comment:
extract out `allowed` array to check for audio or video permissions

Reply:
done in commit - 49d838f10f87eb9c2da9427297afa38cd9b0dcf7